stable/sematext-docker-agent/values.yaml

```diff
@@ -0,0 +1,24 @@
+image:
+  repository: sematext/sematext-agent-docker
+  # For all available tags check here https://hub.docker.com/r/sematext/sematext-agent-docker/tags/
+  tag: 1.31.48
+  pullPolicy: Always
```

Comment:
Per the [review guidelines](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#configuration) can you set the pullPolicy to IfNotPresent. Since this is using tags it shouldn't need to pull again if the tag is present.